content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -41,11 +45,14 @@ some are clearly not very interesting (for example, multiple mesh in a single cl
 
 In a production deployment involving multiple clusters, the deployment may use a
 mix of patterns. For example, having more than one control plane is recommended for HA,
-but you could achieve this for a 3 cluster deployment by deploying 2 clusters with
-a single shared control plane and then adding the third cluster with a second
-control plane in a different network. All three clusters could then be configured
-to share both control planes so that all the clusters have 2 sources of control
-to ensure HA.
+and you can achieve this for a 3 cluster deployment with the following steps:
+
+1. Deploy a {{< gloss >}}primary cluster{{< /gloss >}}
+1. Deploy a {{< gloss >}}remote cluster{{< /gloss >}} on the same network
+1. Deploy a primary cluster in a different network.
```

Comment:
Do you mean cluster creation or Istio control plane creation in steps 1-3? I'm a little confused on the purpose of this page and whether we're describing deployment models of Istio or how to provision your clusters. Could you help clarify?

Reply:
If steps 1-3 are referring to how to setup Istio, then I think the wording needs to change a bit. "Deploy a primary cluster" isn't quite accurate. The cluster(s) already exist. We're deploying/configuring Istio in those clusters to serve the role of a primary/remote cluster in a service mesh.

Reply:
Agreed, we should say something like "Configure a primary cluster", etc. 

I think having this specific example is a bit confusing and misleading here. This sort of content is better left for a best practices guide. Maybe we should just leave it out here?

Reply:
@ayj @nmittler This list will link to the new guides we're introducing. We are not necessarily talking about just configuring either and giving readers an example of a situation where they might need this information is critically important. If not this example, which one would you say is better? Leaving out the info would greatly decrease the quality and effectiveness of the document for readers seeking more opinionated guidance.

Reply:
That's kinda my point ... this is not the right doc for opinionated guidance. That is the responsibility of a best practices guide, which will be written once we've documented all the pieces that we need for it.

The problem with adding this list here is that it reads like it is the best practices, but that is premature information at this point.